### ISSUE TITLE
Question: Make Latex execution stop when an error is encountered

### ISSUE THREAD

**Issue by bergercookie** [kmorel/UseLATEX#24]
When running the xelatex command from the command line it seems to stop at the first encountered error. However this doesn't seem to be the case when running ```cmake .. && make``` using the UseLatex.cmake module in which case execution runs regardless of the error and finally reports if the overall execution was successful. Can this behavior be modified?

Thanks in advance,

**Comment by bergercookie** [comment]
Ah, found it.
This seems to be due to the PDFLATEX_COMPILER_FLAG cmake variable being set by default to "-interaction=nonstopmode". If I run *latex manually with the -interaction=batchmode -file-line-error options, I get the desired effect.

P.S. I think the effect of this flag should be documented in the UseLatex documentation PDF.

**Comment by kmorel** [comment]
I like the options you proposed a lot better than the ones I was using. I change the default options to `-interaction=batchmode -file-line-error`. Because batchmode also suppress warnings that are usually indicative of a malformed document, I also added a script to the build that greps the log file for those warnings.

I also added a section to the FAQ in the documentation about the default LaTeX options.
